src/test/java/uk/gov/justice/probation/courtcasematcher/health/HealthCheckTest.java

```diff
@@ -33,6 +35,9 @@
     @LocalServerPort
     private int port;
 
+    @MockBean
+    private SqsAdminService sqsAdminService;
```

Comment:
Why is this needed?

Reply:
This test runs with "test" profile and SqsMessagingConfig is configured to load when profile is not "test" so SqsAdminService cannot be constructed as it has beans coming from SqsMessagingConfig.

Reply:
Moved to common config